src/3p-frame.js

```diff
@@ -102,16 +102,19 @@ export function getIframe(parentWindow, element, opt_type) {
 }
 
 /**
- * Allows listening for message from the iframe.
+ * Allows listening for message from the iframe. Returns an unlisten
+ * function to remove the listener.
+ *
  * @param {!Element} iframe
  * @param {string} typeOfMessage
  * @param {function(!Object)} callback Called when a message of this type
  *     arrives for this iframe.
+ * @return {function()}
  */
 export function listen(iframe, typeOfMessage, callback) {
```

Comment:
yes, i thought about that. wanted to do it in a separate PR if thats ok with you. since we want to do a release once this bug is fixed.

https://github.com/ampproject/amphtml/issues/861 (create listenOnce helper)